Sources/EpoxyCollectionView/CollectionView/ReusableViews/CollectionViewCell.swift

```diff
@@ -49,6 +49,9 @@ public final class CollectionViewCell: UICollectionViewCell, ItemCellView {
     normalViewBackgroundColor = view.backgroundColor
 
     view.translatesAutoresizingMaskIntoConstraints = false
+    // Use the existing content view size so that we don't have to wait for auto layout to give this
+    // view an initial size.
+    view.bounds = contentView.bounds
```

Comment:
Also worth nothing this is similar to what we do inside the EpoxySwiftUIHostingView itself
https://github.com/airbnb/epoxy-ios/blob/master/Sources/EpoxyCore/SwiftUI/EpoxySwiftUIHostingView.swift#L296-L298

Reply:
Makes sense, I've seen this type of change improve things in the future. However just to be safe let's put this behind a runtime flag in the `CollectionViewConfiguration` so we can disable it if it causes some unforeseen issues